attivita/viste.py

```diff
@@ -155,6 +155,14 @@ def attivita_organizza(request, me):
         attivita.estensione = attivita.sede.comitato
         attivita.save()
 
+        # Crea gruppo per questa specifica attività se la casella viene selezionata.
+        crea_gruppo = modulo.cleaned_data['gruppo']
+        if crea_gruppo:
+            area = Area.objects.get(id=attivita.area_id)
```

Comment:
La round-trip al database non e' necessaria. Puoi sostituire
```python
area = Area.objects.get(id=attivita.area_id)
```
con, piu' semplicemente,
```python
area = attivita.area
```

In generale, tutti i campi di tipo [`django.db.models.ForeignKey`](https://docs.djangoproject.com/en/2.0/ref/models/fields/#django.db.models.ForeignKey) ti forniscono un oggetto a cui puoi accedere direttamente.